cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithLocallyBuildCordappAndConfig.gradle

```diff
@@ -34,6 +37,9 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: [jar]) {
         notary = [validating : true]
         p2pPort 10002
         rpcPort 10003
+        rpcSettings {
+            adminAddress "localhost:10004"
+        }
         cordapp {
```

Comment:
Use `projectCordapp` as your locally-built CorDapp.

Reply:
Fixed in c7d6f965232739388a2819c576d43aa92879dc99